docs/builds/guides/frameworks/vuejs.md

```diff
@@ -80,6 +89,19 @@ const app = new Vue( {
 } );
 ```
 
+```js
+// Only for Vue 3.x
+const app = Vue.createApp( {
+	data() {
+		editor: ClassicEditor,
+		editorData: '<p>Content of the editor.</p>',
+		editorConfig: {
+			// The configuration of the editor.
+		}
+	}
+} ).use( CKEditor( Vue ) ).mount( '#app' );
```

Comment:
If [this line](https://github.com/ckeditor/ckeditor5/pull/8289/files#diff-b2ab47c3d98e0c613d60703d6ef661ade227658ea9062e1e0378cc853ad17904R59) explains how to enable the component globally, I think this `.use()` here is unnecessary, right?

Reply:
The documentation has been rewritten.